packages/vanilla-bundle/src/components/__tests__/slick-vanilla-grid.spec.ts

```diff
@@ -1680,7 +1663,7 @@ describe('Slick-Vanilla-Grid-Bundle Component instantiated via Constructor', ()
         component.gridOptions = { enablePagination: false, showCustomFooter: true };
         component.initialization(divContainer, slickEventHandler);
         const footerSpy = jest.spyOn(component.slickFooter, 'metrics', 'set');
-        mockDataView.onRowCountChanged.notify({ first: 'John', itemCount: 0, current: 0, previous: 1 });
+        mockDataView.onRowsOrCountChanged.notify({ first: 'John', itemCount: 0, current: 0, previous: 1 });
```

Comment:
`current` should be `currentRowCount` with `onRowsOrCountChanged` maybe the Type wasn't detected properly in the test